lib/modified_cops.yml

```diff
@@ -20,3 +20,7 @@ Style/StringLiterals:
   Description: Checks if uses of quotes match the configured preference.
   Enabled: true
   EnforcedStyle: double_quotes
+
+Metrics/BlockLength:
```

Comment:
We can't exclude folders from mad rubocop. The gem can't assume the folder structure of the project. I think this cop should just be turned off.